apps/arweave/src/ar_peers.erl

```diff
@@ -904,13 +950,123 @@ store_peers() ->
 					[],
 					?MODULE
 				),
-			?LOG_INFO([{event, store_peers}, {total, Total}, {records, length(Records)}]),
-			ar_storage:write_term(peers, {Total, Records})
+			Tags = ets:foldl(fun ({{ar_tags, _, _, _}, _} = Tag, Acc) ->
+						[Tag|Acc];
+					     (_, Acc) -> Acc
+					end, [], ?MODULE),
+			?LOG_INFO([{event, store_peers}
+				   , {total, Total}
+				   , {records, length(Records)}
+				   , {tags, length(Tags)}]),
+			ar_storage:write_term(peers, {Total, Records, Tags})
```

Comment:
How are the tags associated with each individual peer? It looks like the tags are being separated from the peers into one long list?

Reply:
Oops, I see. Nvm.